src/mixins/common-mixin.ts

```diff
@@ -39,8 +39,8 @@ export default class CommonMixin extends Vue {
   }
 
   /**
-   * Identifies the first invalid flag and scrolls to the component
-   * @param validatorFlags The component valid flags
+   * Identifies the first invalid flag and scrolls to the component.
+   * @param validatorFlags the component valid flags
```

Comment:
😁

Reply:
If you hover over the method name, it should display nicely.